andela/boondocks-bn-backend
# #169257418 Users should be able to mark all notification as read

#### What does this PR do?
Users should be to mark all notification as read
#### Description of Task to be completed?
- [X] added service to fetch all notifications not read
- [X] added the controller to mark all notifications as read
#### How should this be manually tested?
- Git clone this repo
 - Run `npm run migrate` for migration
 - Run `npx sequelize-cli db:seed:all` to populate the tables.
 - Run `npm run watch` to start the server
- Use postman to login the user and manager in the database on `localhost:3000/api/v1/auth/signin` 
- Mark all your current notifications by using route `localhost:3000/api/v1/markAsRead`
- Confirm by checking the status from route `localhost:3000/api/v1/notification`
#### Any background context you want to provide?
N\A
#### What are the relevant pivotal tracker stories?
[#169257418](https://www.pivotaltracker.com/story/show/169257418)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/52410156/70694223-c35dde80-1cc7-11ea-8026-10a22fbf42bb.png)

![image](https://user-images.githubusercontent.com/52410156/70694268-dcff2600-1cc7-11ea-8e9d-991eab03a471.png)

![image](https://user-images.githubusercontent.com/52410156/70694303-f7390400-1cc7-11ea-94d6-4c1980e35db5.png)

#### Questions:
